src/components/forms/EditForm.jsx

```diff
@@ -46,7 +46,7 @@ const CustomToolbar = (props) => {
   const refresh = useRefresh();
   const redirect = useRedirect();
   const handleClick = () => {
-    let redirectPath = GetRedirectPath(resource, data);
+    const redirectPath = resource === "employees" ? "/employees" : GetRedirectPath(resource, data);
```

Comment:
why this split here? GetRedirectPath methos has a option to handle "employees" resources